src/Configuration/src/Placeholder/PlaceholderResolverExtensions.cs

```diff
@@ -9,98 +9,178 @@
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 using Steeltoe.Common;
 
 namespace Steeltoe.Extensions.Configuration.Placeholder;
 
 public static class PlaceholderResolverExtensions
 {
     /// <summary>
-    /// Create a new <see cref="IConfiguration" /> using a <see cref="PlaceholderResolverProvider" /> which wraps the provided <see cref="IConfiguration" />.
-    /// The new configuration will then be used to replace the current <see cref="IConfiguration" /> in the service container. All subsequent requests for a
-    /// <see cref="IConfiguration" /> will return the newly created one providing placeholder resolution.
+    /// Creates a new <see cref="IConfiguration" /> using a <see cref="PlaceholderResolverProvider" /> which wraps the provided <see cref="IConfiguration" />
+    /// . The new configuration will then be used to replace the current <see cref="IConfiguration" /> in the service container. All subsequent requests for
+    /// a <see cref="IConfiguration" /> will return the newly created one providing placeholder resolution.
```

Comment:
```suggestion
    /// a <see cref="IConfiguration" /> will return the newly created <see cref="IConfiguration" /> with placeholder resolution.
```
I think this is what @ccheetham is getting at